sunbird/common_config/config.ini

```diff
@@ -1,39 +1,46 @@
 [pre-prod]
-host = # Add host Ip address followed by a slash / 
-internal_kong_ip = # Add Internal kong Ip address followed by a slash / 
-keyclockapibody = 
+host = http://localhost:3569/
+internal_kong_ip = http://localhost:3569/
```

Comment:
since both are using interface service url , delete this and keep only one. make relevant change in the script